app/src/androidTest/java/org/simple/clinic/TestClinicApp.kt

```diff
@@ -81,7 +82,7 @@ class TestClinicApp : ClinicApp() {
                 SyncConfig(
                     frequency = Duration.ofMinutes(16),
                     backOffDelay = Duration.ofMinutes(5),
-                    batchSize = 10))
+                    batchSizeEnum = BatchSize.VERY_SMALL))
```

Comment:
`batchSizeEnum` -> `batchSize`?

Reply:
Closing this since it seems to be a duplicate of the other comment.